src/components/navigation/NavLink.tsx

```diff
@@ -0,0 +1,24 @@
+import { LinkProps } from "next/link";
+import Link from "next/link";
+import clsx from "clsx";
+import { ReactNode } from "react";
+
+interface NavLinkProps extends LinkProps {
+  children?: ReactNode;
+  name: string;
+  isActive?: boolean;
+  onClick?: () => void;
+}
```

Comment:
`children`, `onClick` already present in LinkProps type.

```suggestion
interface NavLinkProps extends LinkProps {
  isActive?: boolean;
}
```